src/test/kotlin/com/boclips/apidocs/HestuDocTests.kt

```diff
@@ -93,4 +97,92 @@ class HestuDocTests : AbstractDocTests() {
             .then()
             .assertThat().statusCode(CoreMatchers.`is`(200))
     }
+
+    @Test
+    @Disabled
+    fun `Get highlight`(restDocumentation: RestDocumentationContextProvider) {
+        val responseFields = responseFields(
+            fieldWithPath("getHighlight").description("Get a given highlight by ID"),
+        )
+
+        val pathParameters = RequestDocumentation.pathParameters(
+            RequestDocumentation.parameterWithName("highlightId").description("ID of the highlight"),
+        )
+
+        RestAssured.given(hestuDocumentationSpec)
+            .filter(
+                document(
+                    "highlight-get",
+                    pathParameters,
+                    responseFields,
+                ),
+            )
+            .filter(
+                RestAssuredRestDocumentationWrapper.document(
+                    "{method-name}",
+                    "Can get a given highlight by ID",
+                    false,
+                    responseFields,
+                ),
+            )
+            .`when`()
+            .get(HIGHLIGHT_PATH, productionVideoId).apply { prettyPrint() }
+            .then()
+            .assertThat().statusCode(CoreMatchers.`is`(200))
+    }
+
+    @Test
+    @Disabled
+    fun `Get highlights`(restDocumentation: RestDocumentationContextProvider) {
+        val responseFields = responseFields(
+            fieldWithPath("getHighlights").description("Get filtered highlights"),
+        )
+
+        // TODO("Review the types, descriptions, and optionality of these params w/ DS)
+        val requestParameters = RequestDocumentation.requestParameters(
+            RequestDocumentation.parameterWithName("threshold")
+                .optional()
+                .description("")
+                .attributes(Attributes.key("type").value("Number")),
+            RequestDocumentation.parameterWithName("size")
+                .optional()
+                .description("")
+                .attributes(Attributes.key("type").value("Number")),
+
+            RequestDocumentation.parameterWithName("query")
+                .optional()
+                .description("")
+                .attributes(Attributes.key("type").value("String")),
+
+            RequestDocumentation.parameterWithName("education_level")
+                .optional()
+                .description("")
+                .attributes(Attributes.key("type").value("String")),
+
+            RequestDocumentation.parameterWithName("max_allowed_clip_overlap")
+                .optional()
+                .description("")
```

Comment:
```suggestion
                .description("If multiple clips come from the same video, this is the maximum they are allowed to overlap as a % (expressed as a decimal e.g. 0.2 is 20%) of the smallest video.").
```

default 0.2

Reply:
note we should change this param

Reply:
If 1 a highlight can be a full segment of another highlight. If 0 no overlap is allowed.